package.json

```
@@ -12,14 +12,21 @@
     "live streaming"
   ],
   "author": "Mux <help@mux.com> (https://mux.com/)",
-  "main": "dist",
+  "main": "./cjs/index.js",
+  "exports": "./esm/index.mjs",
```

should `exports` be more explicit?
```suggestion
  "exports": {
    "import": "./esm/index.mjs",
    "require": "./cjs/index.js"
  },
```

---

I thought that if `main` was specified, CJS ignored `exports`.

---

welp this is all terrible